src/plugins/customdata.ts

```diff
@@ -0,0 +1,28 @@
+// Allows data sent via. normal DOM event handlers to be logged alongside Clarity data by listening to the
+// custom DOM event "claritydata".
+import { addEvent, bind } from "./../core";
+
+// Events coming from normal DOM will have additional data in details property
+interface ICustomEvent extends Event {
+    detail: any;
```

Comment:
The typing here is weak. Is there any data contract at all? How would I distinguish different kinds of custom events? At east indicate the type of event.